### ISSUE TITLE
Integral graph flashes when first used

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/calculus-grapher#323]
**Test device**
Samsung
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/924
On the Advanced screen, the first time you switch from the derivative graph to the integral graph the integral graph flashes noticeably. This doesn't happen again after the first time, even after using the reset all button. You need to Refresh the page.
**Steps to reproduce**
1. Refresh the sim if you've been using it
2. Go to the advanced screen
3. Switch to the integral graph

**Visuals**

https://user-images.githubusercontent.com/41024075/228621383-13069797-4842-4365-9f0b-464311244262.MOV

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Calculus Grapher‬
URL: https://phet-dev.colorado.edu/html/calculus-grapher/1.0.0-rc.1/phet/calculus-grapher_all_phet.html
Version: 1.0.0-rc.1 2023-03-28 17:54:45 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36
Language: en-US
Window: 1536x656
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 4096
Texture: size: 8192 imageUnits: 32 (vertex: 32, combined: 64)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by veillette** [comment]
I am able to reproduce the visual artefact on Win 10, with chrome 111.0.5563.147.
However, I am unable to reproduce it  on Win 10, with firefox 111.0.1.

**Comment by KatieWoe** [comment]
I do see this on Win 11 Firefox

**Comment by veillette** [comment]
I stand corrected. I observed it on Win10 firefox as well.

**Comment by veillette** [comment]
I should note that the artefact is also present on the fourth screen with the second derivative graph.

**Comment by pixelzoom** [comment]
Reproduced on macOS 13.2.1 + Chrome 111.0.5563.146.

**Comment by pixelzoom** [comment]
Fixed in the above commit. @veillette and @KatieWoe could you please check in master on your test platforms?  Report back to me, and assign me the issue for cherry-picking.

**Comment by pixelzoom** [comment]
Fixed in the above commit. @veillette and @KatieWoe could you please check in master on your test platforms? Report back to me, and assign me the issue for cherry-picking.

**Comment by veillette** [comment]
Yes the commit addresses the issue on master (tested on Win 10, Chrome).

**Comment by veillette** [comment]
Reassigning to @pixelzoom for cherry-picking.

**Comment by pixelzoom** [comment]
Cherry-pick applied, ready for verification in the next RC test.

**Comment by pixelzoom** [comment]
To verify in the next RC:

1. Go to the Advanced screen.
2. Press the radio button labeled with the integral symbol.
3. At the end of the animation, verify that the integral (top) graph does not flash as it fades in.

4. Go to the Lab screen.
5. Press the radio button labeled _f''(x)_.
6. At the end of the animation, verify that the second derivative (bottom) graph does not flash as it fades in.
